corehq/apps/analytics/templates/analytics/initial/drift.html

```diff
@@ -1,7 +1,13 @@
 {% load hq_shared_tags %}
-{% if is_saas_environment and not request|toggle_enabled:"USER_TESTING_SIMPLIFY" %}
-  {% if not request.user.is_authenticated or request.couch_user.days_since_created < 31 %}
-    {# This was already publicly visible, best practice: todo get into settings #}
+
+{# TODO: API id was already publicly visible, but it should be moved into settings #}
+{% if ANALYTICS_CONFIG.DEBUG %}
+    {# Developer is working on analytics #}
+    {% initial_analytics_data 'drift.apiId' '7456r8nz2u34' %}
+{% elif not is_saas_environment %}
+    {# Don't include: this is an enterprise environment #}
```

Comment:
Inside this elif there is only a comment. Does that work well in django templates? I'm thinking of the following case where python would throw a syntax error

```
if x:
   # blah
elif y:
   do something
```

Reply:
yeah, templates are okay with this syntax (tested)